lib/knex_cleaner.js

```diff
@@ -1,27 +1,49 @@
-'use strict';
+"use strict";
 
-var BPromise = require('bluebird');
-var _ = require('lodash');
+var BPromise = require("bluebird");
+var _ = require("lodash");
 
-var knexTables = require('../lib/knex_tables');
+var knexTables = require("../lib/knex_tables");
+var fs = require("fs");
+var path = require("path");
+var NODE_ENV = process.env.NODE_ENV || "development";
 
 var DefaultOptions = {
-  mode: 'truncate',    // Can be ['truncate', 'delete']
-  ignoreTables: []     // List of tables to not delete
+  mode: "truncate", // Can be ['truncate', 'delete']
+  ignoreTables: [] // List of tables to not delete
 };
 
 function clean(knex, passedInOptions) {
+  var knexConfigPath = passedInOptions.knexConfigPath
+    ? passedInOptions.knexConfigPath
+    : path.resolve(process.cwd(), "knexfile.js");
+
+  if (!fs.existsSync(knexConfigPath)) {
+    throw `Unable to locate knex config at ${knexConfigPath}`;
+  }
+
+  var knexConfig = require(knexConfigPath);
+  var knexMigrationsTable =
```

Comment:
This could probably be cleaned up by using[ lodash's `get` function](https://lodash.com/docs/4.17.4#get)